frontend/src/libs/enums/app-route.enum.ts

```diff
@@ -2,6 +2,7 @@ const AppRoute = {
   ROOT: '/',
   SIGN_IN: '/sign-in',
   SIGN_UP: '/sign-up',
+  SURVEY: '/survey',
```

Comment:
```suggestion
  SURVEY: '/sign-up/survey',
```
Also here

Reply:
I was getting an error and background wasn't displayed with this route while unauthenticated user redirects from survey page, so I leave it as it was

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/71706808-faa4-46fa-a1ca-d1e2ef09e19f)

Reply:
it should be sign-up/survey, what error did you get?

Reply:
@Liza-Veis After redirecting from `/survey` to sign in everything works fine, however after redirecting from `/sign-up/survey` css by using `url()` in background image trying to fetch it from `http://localhost:3000/sign-up/images/survey-1440x1024.png` instead of `http://localhost:3000/images/survey-1440x1024.png` which leads to absence of the background. I'm trying to figure out the solution but still didn't find it

Reply:
@Dmytro-Kucherenko inside of `url()` try starting the image url with '/'. For example `url('/images/...')`

Reply:
@fshabanov Yeah, it helped, thanks!